base/applications/mspaint/main.cpp

```diff
@@ -179,13 +179,28 @@ HWND CMainWindow::DoCreate()
     return Create(HWND_DESKTOP, rc, strTitle, WS_OVERLAPPEDWINDOW, WS_EX_ACCEPTFILES);
 }
 
+#ifdef _DEBUG
+void CheckHandleLeaks(void)
+{
+#if (WINVER >= 0x0500)
```

Comment:
Not sure this is important. Everything we implement is of NT >= 5.

Reply:
Deleted.